graphql/schemabuilder/pagination.go

```diff
@@ -298,23 +333,47 @@ func (o *Object) PaginateFieldFunc(name string, f interface{}) {
 		})
```

Comment:
can we update the comment for `PaginateFieldFunc` to talk about the pagination modes available after this PR?

Reply:
done